ext/opcache/zend_accelerator_module.c

```diff
@@ -170,12 +170,7 @@ static ZEND_INI_MH(OnUpdateFileCache)
 
 		    if (!IS_ABSOLUTE_PATH(ZSTR_VAL(new_value), ZSTR_LEN(new_value)) ||
 			    zend_stat(ZSTR_VAL(new_value), &buf) != 0 ||
-			    !S_ISDIR(buf.st_mode) ||
-#ifndef ZEND_WIN32
-				access(ZSTR_VAL(new_value), R_OK | W_OK | X_OK) != 0) {
-#else
-				_access(ZSTR_VAL(new_value), 06) != 0) {
-#endif
+			    !S_ISDIR(buf.st_mode)) {
```

Comment:
Move all this checks into post startup. May be ``OnUpdateFileCache`` won't be necessary at all.
No need to squash.